packages/text-input/src/styles.tsx

```diff
@@ -65,6 +65,7 @@ export const getTextInputStyle = (appearance?: TextInputAppearanceType) => (_: {
         backgroundColor: 'transparent',
         boxShadow: colors.input.outline.boxShadow.default,
         ':focus': {
```

Comment:
You have moved textarea style to a div. Div doesn't have focus state unless it has tabIndex attribute - but there's other solution, you can use [:focus-within](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within) to style container if any children has focus.